### ISSUE TITLE
Things.sqlite3 move in Things 3.12.6

### ISSUE THREAD

**Issue by brokosz** [AlexanderWillner/things.sh#30]
Cultured Code moved the Things db in 3.12.6 to `~/Library/Group Containers`. Subsequently, things.sh fails locating the file:
```bash
$ things.sh today
Error: file is not a database
ERROR: line 1 - command 'eval "${!method}"' exited with status: 26.
ERROR: In ::queryToday::invokePlugin::parse::main::main called at line 48.
DEBUG: From function ::queryToday::invokePlugin::parse::main::main (line 48).
```

replacing
`readonly DEFAULT_DB=~/Library/Containers/com.culturedcode.ThingsMac/Data/Library/Application\ Support/Cultured\ Code/Things/Things.sqlite3`
with
`readonly DEFAULT_DB="/Library/Group Containers/JLMPQHK86H.com.culturedcode.ThingsMac/Things.sqlite3"`
restored access for me, though I don't know if the `JLMPQHK86H` part is unique per user...

**Comment by AlexanderWillner** [comment]
Thank you. Seems the `JLMPQHK86H` is not unique per user.

**Comment by AlexanderWillner** [comment]
@brokosz As I currently have only the beta installed: are you sure it's `readonly DEFAULT_DB="/Library/Group Containers/JLMPQHK86H.com.culturedcode.ThingsMac/Things.sqlite3"` and not `readonly DEFAULT_DB="~/Library/Group Containers/JLMPQHK86H.com.culturedcode.ThingsMac/Things.sqlite3"`?

**Comment by brokosz** [comment]
Sorry @AlexanderWillner  - I had to go check... for me it actually only works with the full path, not the `~` path. so mine is actually `/Users/<USERNAME>/Library/Group Containers/JLMPQHK86H.com.culturedcode.ThingsMac/Things.sqlite3`. I'm sure there's an env var I'm missing to get it work off the local path

**Comment by AlexanderWillner** [comment]
Ok, thanks. Will push an update soon.

**Comment by AlexanderWillner** [comment]
Fixed in release 2.9
